gpu/test_gpu.py

```diff
@@ -16,6 +18,10 @@ def verify_instance_gpu_agent(self, name):
     self.assert_instance_command(
         name, "systemctl status gpu-utilization-agent.service")
 
+  def verify_instance_cudnn(self, name):
+    self.assert_instance_command(
+        name, "sudo find /usr -iname libcudnn.so | grep .")
```

Comment:
What does `grep .` mean?

Reply:
`grep .` is checking if there's any output from `find`, as `find` always returns exit code zero.

Reply:
Maybe something like this will be more explicit:
```
[[ -n $(sudo find /usr -iname libcudnn.so | wc -l) ]]
```

Reply:
Ack, fixed (with a slight tweak)